tools/gff2JSON/gff-to-json.xml

```diff
@@ -0,0 +1,64 @@
+<tool id="gff-to-JSON_wrapper" name="GFF to JSON" version="0.1.1">
+    <description>
+        Generate a JSON formated Gene file from GFF format
+    </description>
+    <command>
+        <![CDATA[
+        perl $__tool_directory__/gff-to-json.pl 
+        ${genome1}:$inputFile 
+        #for $q in $queries
+            ${q.genome2}:${q.input2}
+        #end for
+        > $output
+        ]]>
+    </command>
+
+    <inputs>
+    <param name="inputFile" type="data" format="gff3"  label="GFF file" help="GFF3 file format" />
+    <param name="genome1" type="text" format="text" label="Genome name" help="Genome name without white space or special character" />
+        <repeat name="queries" title="Series">
+            <param name="input2" type="data" format="gff3" label="Dataset"/>
+            <param name="genome2" type="text" format="text" label="Genome name" help="Genome name without white space or special character" />
+        </repeat>
```

Comment:
You can add `min="1"` as an attribute of the `<repeat>` instead of using this trick.